src/pages/console/tutorial/bindui.mdx

```diff
@@ -1,17 +1,17 @@
 export const meta = {
   title: `Connect UI to data`,
-  description: `Figma to React code with Amplify Studio`,
+  description: `Figma to React code with Amplify Studio`
 };
 
-Studio also allows you to bind UI elements to real backend data. Add properties to components, then reference the properties in your child UI elements. To get started, navigate into a UI component by choosing the **Configure** button in the **UI Library**. For this example, you'll use the *CardB* component.
+Studio also allows you to bind UI elements to real backend data. Add properties to components, then reference the properties in your child UI elements. In [Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app), navigate into a UI component by choosing the **Configure** button in the **UI Library**. For this example, you'll use the _CardB_ component.
```

Comment:
This one works, but I might write it as "To get started, first, [login to Amplify Studio](/console/adminui/start/#logging-in-and-creating-an-app), then navigate into a UI component by choosing the **Configure** button in the **UI Library**.